geonode/harvesting/harvesters/geonode.py

```diff
@@ -393,6 +424,20 @@ def _get_resource_list_params(
         }
         if self.resource_title_filter is not None:
             result["title__icontains"] = self.resource_title_filter
+        if self.start_date_filter is not None:
+            start_date = dateutil.parser.parse(self.start_date_filter)
+            result["date__gte"] = start_date.astimezone(
+                pytz.utc).replace(microsecond=0).isoformat().split('+')[0] + 'Z'
+        if self.end_date_filter is not None:
+            end_date = dateutil.parser.parse(self.end_date_filter)
+            result["date__lte"] = end_date.astimezone(
+                pytz.utc).replace(microsecond=0).isoformat().split('+')[0] + 'Z'
```

Comment:
Please use Python's built in UTC support instead of pytz here

`datetime.timezone.utc`

Reply:
updated @ricardogsilva